common/djangoapps/track/backends/oli.py

```diff
@@ -0,0 +1,127 @@
+"""
+OLI analytics service event tracker backend.
+"""
+
+from __future__ import absolute_import
+
+import json
+import logging
+
+from requests_oauthlib import OAuth1Session
+from urlparse import urljoin
```

Comment:
Since this is part of Python's stdlib, it should be moved to the 2nd stanza, below `import logging`,